### PR TITLE
Navbar section

### PR DESCRIPTION
**Pull request by zaman2031**

### CODE REVIEW

File: my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import NavBar from './components/Navbar';
+import Profile from './components/Profile';
+
+function App() {
+  return (
+    <div className="App">
+       <NavBar />
+       {/* <Profile /> */}
```
Comment (omair445) [inline my-app/src/App.js]: Remove all the commented out code from your PR.

File: my-app/src/components/Navbar.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  float:right;
+}
+
+.logo img {
+  float: left;
+  margin: 10px;
+  width: 7%;
+}
+
+
```
Comment (omair445) [inline my-app/src/components/Navbar.scss]: Remove extra spaces.

File: my-app/src/components/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (omair445) [inline my-app/src/components/Profile.jsx]: Remove this component file from your PR as it is not related to the PR.

File: my-app/src/index.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (omair445) [inline my-app/src/index.css]: Are all these fonts families are using in the project?

File: my-app/src/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```
Comment (omair445) [inline my-app/src/index.js]: Please explain the use of the method.

**Review by omair445 — CHANGES_REQUESTED** [review]

**Comment by omair445** [comment]
@zaman2031 
Remove these files from your PR.
package-lock
manifest
robot.txt
app.test
setuptest

File: my-app/src/App.js (hunk 1 above)
Comment (zaman2031) [reply my-app/src/App.js]: removed

File: my-app/src/components/Navbar.scss (hunk 2 above)
Comment (zaman2031) [reply my-app/src/components/Navbar.scss]: removed

File: my-app/src/index.css (hunk 4 above)
Comment (zaman2031) [reply my-app/src/index.css]: No

File: my-app/src/components/Profile.jsx (hunk 3 above)
Comment (zaman2031) [reply my-app/src/components/Profile.jsx]: removed

File: my-app/src/index.js (hunk 5 above)
Comment (zaman2031) [reply my-app/src/index.js]: Need more Searching I know little bit about this function.

File: my-app/public/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
```
Comment (MuneebSVS) [inline my-app/public/index.html]: Commented out code still exist. I have mentioned you to specifically look into whole PR and remove it.

File: my-app/src/components/Navbar.jsx [hunk 7]
```diff
@@ -1,27 +1,24 @@
 import React from 'react';
-import logo from '../Logos/logo.png'
-import newLogo from '../Logos/logo.png';
```
Comment (nb-1231) [inline my-app/src/components/Navbar.jsx]: use assets folder for images

File: my-app/src/components/Navbar.jsx [hunk 8]
```diff
@@ -1,27 +1,24 @@
...
+import './Navbar.scss';
 
 const Navbar = () => {
     return (
-        <div className='container'>
-            <div className="logo">
-                <img src={logo} alt="This is image" />
+            <div className='main-container'>
+                <div className='wecare-logo'>
+                    <img src={logo} alt="" />
```
Comment (nb-1231) [inline my-app/src/components/Navbar.jsx]: fill out the alt property

File: my-app/src/components/Navbar.jsx [hunk 9]
```diff
@@ -1,27 +1,24 @@
...
+                </div>
+                <nav>
+                    <ul>
+                        <li>Job search</li>
+                        <li>be on one's mind!</li>
+                        <li>message</li>
+                        <li className='list-item'>My page</li>
+                        <li className='N-character'>N</li>
```
Comment (nb-1231) [inline my-app/src/components/Navbar.jsx]: use a better name for classes

File: my-app/src/components/Navbar.jsx [hunk 10]
```diff
@@ -1,27 +1,24 @@
...
-                    <li className='list-item'>My page</li>
-                    <li className='list-image'>
-                    <img src={newLogo} alt="" />
-                    </li>
-                </ul>
-                <button className='Logout-button'>Logout</button>
-            </nav>
-        </div>
     );
 }
```
Comment (nb-1231) [inline my-app/src/components/Navbar.jsx:22]: remove extra spaces

**Review by nb-1231 — CHANGES_REQUESTED** [review]

**Comment by nb-1231** [comment]
navbar content must be aligned

File: my-app/src/components/Navbar.jsx (hunk 7 above)
Comment (zaman2031) [reply my-app/src/components/Navbar.jsx]: redolved

File: my-app/src/components/Navbar.jsx (hunk 8 above)
Comment (zaman2031) [reply my-app/src/components/Navbar.jsx]: resolved

File: my-app/src/components/Navbar.jsx (hunk 9 above)
Comment (zaman2031) [reply my-app/src/components/Navbar.jsx]: resolved

File: my-app/src/components/Navbar.jsx (hunk 10 above)
Comment (zaman2031) [reply my-app/src/components/Navbar.jsx:22]: resolved

File: my-app/src/index.css (hunk 4 above)
Comment (MuneebSVS) [reply my-app/src/index.css]: Remove the extra fonts.

File: my-app/src/index.js (hunk 5 above)
Comment (MuneebSVS) [reply my-app/src/index.js]: Have you search for this method.

**Review by MuneebSVS — CHANGES_REQUESTED** [review]

File: my-app/public/index.html (hunk 6 above)
Comment (zaman2031) [reply my-app/public/index.html]: This come by default with create react app . I think whole pr means that files on which i am working . Sorry for that I am removing this and push again thanks.

Comment (MuneebSVS) [reply my-app/public/index.html]: If the file are showing in your PR its your responsibility either it is created by you or it is generate by default.
You need to take a closer look at you code while pushing it not just use "git add ." .

**Comment by MuneebSVS** [comment]
Your PR now contain the changes for the profile, info section etc that shouldn't be there. PR should only contain navbar changes.

**Review by MuneebSVS — APPROVED** [review]

**Comment by zaman2031** [comment]
Next time i will be careful about my previous mistakes .

**Comment by zaman2031** [comment]
ok sir

**Pull request merged**